src/amber/cli/templates/app/src/views/layouts/application.ecr.ecr

```diff
@@ -19,6 +19,9 @@
         <ul class="navbar-nav mr-auto">
           <%="<"%>%= render(partial: "layouts/_nav.ecr") %>
         </ul>
+        <ul class="navbar-nav">
+          <%="<"%>%= render(partial: "layouts/_session.ecr") %>
+        </ul>
```

Comment:
can we inject this in the auth generator instead?  I don't want to polluted the initial `app` with stuff from `auth`.